frontend/src/scenes/experiments/MetricsView/DeltaChart.tsx

```diff
@@ -606,14 +606,17 @@ export function DeltaChart({
                                             <span className="font-semibold">
                                                 {(() => {
                                                     try {
-                                                        const detail = JSON.parse(error.detail)
```

Comment:
Not sure when `error.detail` was ever a string. It was a proper object in my testing.